### PR TITLE
CFE-2850 Added documentation entry for string_replace

### PR DESCRIPTION
**Pull request by karlhto**
Ticket: CFE-2850
Changelog: Title

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

File: reference/functions/string_replace.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Example:
+
+[%CFEngine_include_snippet(string_replace.cf, #\+begin_src cfengine3, .*end_src)%]
+
+Output:
+
+[%CFEngine_include_snippet(string_replace.cf, #\+begin_src\s+example_output\s*, .*end_src)%]
+
+History: Introduced in 3.12.0 (2018) as a simpler version of `regex_replace()`
```
Comment (vpodzime) [inline reference/functions/string_replace.markdown]: This should be *3.12.1*.

File: reference/functions/string_replace.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Reads a string from left to right, replacing the occurences of the second
+argument with the third argument in order.
+
+All characters in the string to replace in, the substring to match for and the
+replacement are read literally. This means that `.`, `*`, `\` and similar
+characters will be read and replaced as they are. If you are looking for more
+advanced replace functionality, check out `regex_replace()`.
+
+Example:
```
Comment (nickanderson) [inline reference/functions/string_replace.markdown]: Example, output, history, and see also should be bolded like Description is above.

Comment (karlhto) [reply reference/functions/string_replace.markdown]: Output doesn't seem to be bolded in a lot of the other reference entries?

Comment (karlhto) [reply reference/functions/string_replace.markdown]: Seems like it is inconsistent. I'll add it anyway!

**Comment by karlhto** [comment]
@nickanderson Does it look fine now?

**Comment by nickanderson** [comment]
Thank you!

**Pull request merged**